### PR TITLE
Update UG to fix general UG bug

### LINKED ISSUE #238: Fix general UG bugs
resolves #211 
resolves #212 
resolves #205

### PR DESCRIPTION
**Pull request by SiTingST**
resolves #238

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -69,13 +69,14 @@ Adds the details of a student to Vax@NUS records. It is mandatory to include the
...
 
+
 * School Residence, to determine if the student lives on campus. This will help NUS campus residences determine the proportion of vaccinated student residents using the `stats` command. 
 
 :information_source: **NOTE** If the School Residence of a student is not specified, the system will default to DOES_NOT_LIVE_ON_CAMPUS and assume that the student does not live on campus.
 
 > For a smooth user experience, please refer to the  please refer to the [Input Formats](#input-formats) section below for more information regarding input formats. That section explains which prefix should be used for each piece of information and how the information should be presented to the program. 
 
-`Add` Command Format: `add n/NAME i/MATRICULATION_NUMBER f/FACULTY p/PHONE_NUMBER e/EMAIL a/ADDRESS s/VACCINATION_STATUS m/MEDICAL_DETAILS r/SCHOOL_RESIDENCE[optional]`
+`Add` Command Format: `add n/NAME i/MATRICULATION_NUMBER f/FACULTY p/PHONE_NUMBER e/EMAIL a/ADDRESS s/VACCINATION_STATUS m/MEDICAL_DETAILS [r/SCHOOL_RESIDENCE]`
```
Comment (yienyoong) [inline docs/UserGuide.md:79]: Sorry, just to clarify, is SCHOOL_RESIDENCE now optional? I may be mistaken but based on my understanding, it is still an optional field?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -343,18 +346,18 @@ Vax@NUS saves your current date in the hard disk automatically after any command
...
 **Delete Student** | `delete MATRICULATION_NUMBER` e.g., `delete A1234567X`
 **Filter Students** | `filter VACCINATION_STATUS`  e.g., `filter vaccinated`, `filter not vaccinated`, <br> `filter FACULTY ` e.g., `filter COM` <br> `filter SCHOOL_RESIDENCE` e.g., `filter RVRC` 
 **View Student Statistics** | `stats FACULTY` e.g., `stats COM` <br> `stats SCHOOL_RESIDENCE` e.g., `stats RC4` <br> `stats NUS` <br> `stats all` 
-**Add Appointment** | `addAppt i/MATRICULATION_NUMBER d/DATE ts/START_TIME` e.g., `addAppt i/A1234567X d/2021-12-13 ts/13:00`
-**Edit Appointment** | `editAppt MATRICULATION_NUMBER d/DATE_YYYY-MM-DD ts/START_TIME_HH:MM` e.g.,` editAppt A1234567X d/2021-12-13 ts/14:00`
-**Delete Appointment** | `deleteAppt MATRICULATION_NUMBER` e.g., `deleteAppt A1234567X`
+**Add <br> Appointment** | `addAppt i/MATRICULATION_NUMBER d/DATE ts/START_TIME` <br> e.g., `addAppt i/A1234567X d/2021-12-13 ts/13:00`
+**Edit Appointment** | `editAppt MATRICULATION_NUMBER d/DATE_YYYY-MM-DD ts/START_TIME_HH:MM` <br> e.g.,` editAppt A1234567X d/2021-12-13 ts/14:00`
```
Comment (yienyoong) [inline docs/UserGuide.md:346]: Just to clarify as well, I am unsure as to the reason for adding '\<br>' for Add Appointment but not for Edit Appointment and Delete Appointment.

**Review by yienyoong — COMMENTED** [review]
I have some queries regarding some changes, however, the rest looks good. Thank you Si Ting!

File: docs/UserGuide.md (hunk 1 above)
Comment (SiTingST) [reply docs/UserGuide.md:79]: Yea, it is still an optional field.. Earlier on in our UG, we state that items  in square bracket are optional so we should stil follow their convection!

File: docs/UserGuide.md (hunk 2 above)
Comment (SiTingST) [reply docs/UserGuide.md:346]: Good observation! I just realized that we might not need the <br> for the commands..

Comment (SiTingST) [reply docs/UserGuide.md:346]: I will fix it now!

File: docs/UserGuide.md (hunk 1 above)
Comment (yienyoong) [reply docs/UserGuide.md:79]: Apologies for the oversight! I understand now.

File: docs/UserGuide.md (hunk 2 above)
Comment (yienyoong) [reply docs/UserGuide.md:346]: Thank you! :)

**Review by yienyoong — APPROVED** [review]
LGTM!

**Pull request merged**